### PR TITLE
fix: bootloader_start return VM_SYS_RQ from VM and stuck on flash ins…

### PR DESCRIPTION
**Pull request by ihormudryy**
…tall

### CODE REVIEW

File: src/bl_install.c [hunk 1]
```diff
@@ -63,12 +57,7 @@ int bootloader_install(FILE *fp)
...
 	u32 remote_crc32 = 0;
 
-	puts("Erasing flash... (takes 1 minute 48 seconds)"); // 108 seconds
-	err = bootloader_erase();
-	if (err != ERR_UNK)
-		return err;
-
-	puts("Downloading...");
+	puts("Erasing and Downloading...");
 	err = bootloader_start(FLASH_START, FLASH_SIZE); // extremely short
```
Comment (JakubVanek) [inline src/bl_install.c:61]: only the comment here needs adjusting

**Comment by JakubVanek** [comment]
Apart from whether this is the best solution for https://github.com/c4ev3/ev3duder/issues/32, this looks good to me.

**Comment by ihormudryy** [comment]
I also don't know if this solution is the best as I am not a C programmer, but it suddenly worked for me with my ubuntu 20.10 and USB 3.1.

**Comment by ihormudryy** [comment]
Worked for me too! 
```
sudo ./ev3duder flash install ~/Mindstorm/1_10.bin 
USB connection established.
Erasing flash... (takes 1 minute 48 seconds)
Downloading...
Progress:   0 %
Progress:   5 %
Progress:  10 %
Progress:  15 %
Progress:  20 %
Progress:  25 %
Progress:  30 %
Progress:  35 %
Progress:  40 %
Progress:  45 %
Progress:  50 %
Progress:  55 %
Progress:  60 %
Progress:  65 %
Progress:  70 %
Progress:  75 %
Progress:  80 %
Progress:  85 %
Progress:  90 %
Progress:  95 %
Progress: 100 %
Calculating flash checksum... (takes 17 seconds)
Success! Local and remote checksums match. Rebooting.
Update mode was exited successfully.

```

Please add in README that the user must pull all the submodules otherwise it's not intuitive why compilation is failing if to clone the clean repo. 
`git submodule update --init --recursive`

**Comment by JakubVanek** [comment]
Superseded by https://github.com/c4ev3/ev3duder/pull/34

**Comment by a3f** [comment]
Thanks you two for taking the time to sort this out :^)

**Pull request closed without merging**